build.js

```diff
@@ -14,28 +14,32 @@
 
 'use strict';
 
-const css = require('@webref/css');
-const fs = require('fs-extra');
-const idl = require('@webref/idl');
-const path = require('path');
-const prettier = require('prettier');
-const WebIDL2 = require('webidl2');
-const YAML = require('yaml');
+import css from '@webref/css';
+import esMain from 'es-main';
+import fs from 'fs-extra';
+import idl from '@webref/idl';
+import path from 'path';
+import prettier from 'prettier';
+import {fileURLToPath} from 'url';
+import {validate} from 'webidl2';
```

Comment:
```suggestion
import * as WebIDL2 from 'webidl2';
```

To avoid a bunch of changes that aren't strictly required. I also think it's clearer in context to have `WebIDL2.validate()` or `WebIDL2.parse()`, not the bare functions which have very generic names.